lib/components/ArtistScreen/artist_screen_content.dart

```diff
@@ -17,66 +17,60 @@ class ArtistScreenContent extends StatefulWidget {
 }
 
 class _ArtistScreenContentState extends State<ArtistScreenContent> {
-  Future<List<BaseItemDto>?>? songs;
-  Future<List<BaseItemDto>?>? albums;
   JellyfinApiHelper jellyfinApiHelper = GetIt.instance<JellyfinApiHelper>();
 
   @override
   Widget build(BuildContext context) {
-    songs ??= jellyfinApiHelper.getItems(
-      parentItem: widget.parent,
-      filters: "Artist=${widget.parent.name}",
-      sortBy: "PlayCount",
-      includeItemTypes: "Audio",
-      isGenres: false,
-    );
+    final futures = Future.wait([
+      jellyfinApiHelper.getItems(
```

Comment:
Maybe add to comments here to say what each request does (request top songs, request albums) (and that it will be index 0/1 in the `futures` variable)